content/css/concepts/floats/floats.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'Floats'
+Description: ''
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Floats'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `float` propertie moves an element with in its parent.
+
+## Syntax
+
+The `float` property takes 5 different values. They are, `left`, `right`, and `none`. The value `left` makes the element float on the left of a parent. The value `right` makes the element float on the right of a parent. The value `none` makes the element not float at all.
```

Comment:
Here you have added 5 values and you have described only 3, could you please add other 2 as well i.e. initial and inherit? with description?